components/canvas/webgl_mode/inprocess.rs

```diff
@@ -27,8 +28,10 @@ use webxr_api::SwapChainId as WebXRSwapChainId;
 pub struct WebGLComm {
     pub webgl_threads: WebGLThreads,
     pub webxr_swap_chains: SwapChains<WebXRSwapChainId>,
+    pub webxr_surface_providers: SurfaceProviders,
```

Comment:
I'd prefer an API that merged these two back together, but that can wait for now.

Reply:
The swap chains and surface providers, you mean?

Reply:
Yes, I suspect that would be tidier, but it's not an urgent change.